node/setup.go

```diff
@@ -727,7 +727,7 @@ func makeNodeInfo(
 	var bcChannel byte
 	switch config.BlockSync.Version {
 	case cfg.BlockSyncV0:
-		bcChannel = byte(bcv0.BlockchainChannel)
+		bcChannel = byte(bcv0.BlockSyncChannel)
 
 	case cfg.BlockSyncV2:
 		bcChannel = bcv2.BlockchainChannel
```

Comment:
We're not worried about V2 right? 🙃

Reply:
I saw some discussions about canceling v2 and also #6755 didn't touch upon it, so I decided not to touch it as well.

Reply:
Yeah totally makes sense. Let's leave it for now.